tests/test_browser.py

```diff
@@ -1681,6 +1690,18 @@ def test_chunked_synchronous_xhr(self):
 
     server = multiprocessing.Process(target=test_chunked_synchronous_xhr_server, args=(True, chunkSize, data, checksum, self.port))
     server.start()
+
+    # block until the server is actually ready
+    for i in range(60):
+      try:
+        urlopen('http://localhost:11111')
```

Comment:
Shouldn't this involve `self.port` in some way?

Reply:
No, `port` is the port of the main test runner, while this 11111 is a special extra port used just in this test, and we run a separate server on it.